planner/core/expression_rewriter.go

```diff
@@ -1276,3 +1280,47 @@ func (er *expressionRewriter) toColumn(v *ast.ColumnName) {
 	}
 	er.err = ErrUnknownColumn.GenWithStackByArgs(v.String(), clauseMsg[er.b.curClause])
 }
+
+func (er *expressionRewriter) evalDefaultExpr(v *ast.DefaultExpr) {
+	col, err := er.schema.FindColumn(v.Name)
```

Comment:
Can we pop and use the column on the stack directly?

Reply:
Seems not, what we want to get here is the default value of the column.

Reply:
I mean we can avoid function call of `FindColumn` by getting the `Column` from stack, but the logics below this line is needed also.

Reply:
hi @eurekaka , when you talk about 'getting the `Column` from stack', do you mean `ctxStack`? I've no idea how to get the column from it. Any advice is appreciated, thank you.

Reply:
Yes, I am referring to `ctxStack`, there should be a `Column` on top of it since expressionRewriter would visit argument of `default()` function first. In line 1312, you have these lines:
```golang
	stkLen := len(er.ctxStack)
	er.ctxStack = er.ctxStack[:stkLen-1]
```
the `er.ctxStack[stkLen-1]` should be the `Column` built for the argument of `default`.

Reply:
I see, thank you!